lib/pages/trade/models/TradePageEvent.dart

```diff
@@ -0,0 +1,65 @@
+import 'package:ax_dapp/service/Controller/Token.dart';
+import 'package:equatable/equatable.dart';
+
+class TradePageEvent extends Equatable {
+  @override
+  List<Object?> get props => [];
+}
+
+class PageRefreshEvent extends TradePageEvent {
+  @override
+  List<Object?> get props => [];
+  PageRefreshEvent();
+}
+
+class NewTokenFromInputEvent extends TradePageEvent {
+  final double tokenInputFromAmount;
+
+  @override
+  List<Object?> get props => [];
```

Comment:
put the class properties as entries in the props array here

Reply:
done